packages/block-editor/src/components/block-canvas/index.js

```diff
@@ -43,6 +44,7 @@ export function ExperimentalBlockCanvas( {
 				__unstableContentRef={ localRef }
 				style={ { height, display: 'flex' } }
 			>
+				<EditorFontsResolver />
```

Comment:
@youknowriad or @ellatrix since you've edited this file before, could you please sense-check how the fonts resolver is being added here?